fetch_news.py

```diff
@@ -156,6 +153,20 @@ def get_mean_embedding(mongo_client = None, connection_string = None, classifica
 
     return m
 
+def delete_old_article_text(mongo_client):
+    candidates_collection = mongo_client['aiidprod'].candidates
+    for article in candidates_collection.find({'text': {'$exists': True}}):
+      article_date = dateutil.parser.parse(
+        article.get('date_published') or 
+        article.get('date_scraped') or 
+        '2023-08-22'
```

Comment:
nit: leave a comment on magic value? e.g. "This is the date pruning of old articles was implemented."